frontend/components/common/Login.tsx

```diff
@@ -9,21 +9,35 @@ function Login() {
           minHeight: "90vh",
           display: "flex",
           justifyContent: "center",
-          flexDirection: "column",
-          textAlign: "center",
-          alignContent: "center",
           alignItems: "center",
-          gap: "20px",
+          gap: "20px"
         }}
       >
+        <style>
+          {`
+          @media (max-width: 600px) {
+            div {
+              flex-direction: column;
+            }
+          `}

```

Comment:
can you move the media query into [Main.module.css](https://github.com/J0SAL/Decentralized-Expense-Tracker/blob/main/frontend/styles/Main.module.css)

Reply:
> can you move the media query into [Main.module.css](https://github.com/J0SAL/Decentralized-Expense-Tracker/blob/main/frontend/styles/Main.module.css)

Ya sure